contracts/cw-admin-factory/src/msg.rs

```diff
@@ -0,0 +1,20 @@
+use cosmwasm_std::Binary;
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
+pub struct InstantiateMsg {}
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
+#[serde(rename_all = "snake_case")]
+pub enum ExecuteMsg {
+    InstantiateContract {
```

Comment:
Could we add a docstring to this message type to explain what it does? Eventually, we'll package up these contracts and publish them as cargo crates and the docs will be nice for folks looking at them.